node/src/main/java/eu/cloudnetservice/node/service/defaults/AbstractService.java

```diff
@@ -394,6 +394,11 @@ public void includeWaitingServiceInclusions() {
         FileUtil.copy(destination, target);
         // we've installed the inclusion successfully
         this.installedInclusions.add(inclusion);
+
+        // we don't want to keep the temporary file if caching is disabled
+        if (!inclusion.cacheFiles()) {
+          FileUtil.delete(destination);
+        }
```

Comment:
I'm wondering: wouldn't it be easier to just download the file into the service directory rather than downloading to a temp dir, copying the file & deleting it?